specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-08-02-preview/managedClusters.json

```diff
@@ -7052,6 +7059,17 @@
       "minLength": 1,
       "maxLength": 36,
       "x-ms-parameter-location": "method"
+    },
+    "AgentPoolWindowsProfile": {
+      "type": "object",
+      "description": "The Windows node's specific profile.",
+      "properties": {
+        "disableOutboundNAT": {
+          "type": "boolean",
+          "title": "Whether each Windows node disables OutboundNAT",
+          "description": "The default value is false. Windows nodes enable OutboundNAT by default. Allow disable OutboundNAT only if the cluster outbound type is NAT Gateway and the Windows agent pool doesn't enable node public IP."
```

Comment:
Allow disabling OutboundNAT?

Reply:
the Windows agent pool doesn't enable node public IP => the Windows agent pool doesn't have node public IP enabled?

Reply:
"Windows nodes enable OutboundNAT by default" may be duplicated of "The default value is false."